_howdoi/choose-my-spark-distribution.adoc

```diff
@@ -0,0 +1,51 @@
+= How do I choose my own Apache Spark distribution?
+:page-layout: howdoi
+:page-menu_entry: How do I?
+
+When you install radanalytics.io, your project is set up to use a pre-determined distribution of Apache Spark. You can choose your own Apache Spark distribution with a few additional steps:
+
+.Prerequisites
+
+. Follow the instructions under link:/howdoi/install-radanalyticsio[How do I install radanalytics.io?]
+
+. Choose an Apache Spark binary tarball like link:https://archive.apache.org/dist/spark/spark-2.3.0/spark-2.3.0-bin-hadoop2.7.tgz[this one]. You can download it or just use the url. Any binary distribution should work as long as it has this basic structure:
+
+* a top-level directory we'll call TOP
+* binaries under TOP/bin
+* jar files under TOP/jars
+* configuration files under TOP/conf
+
+.Procedure
+
+. Get the `rad-image` script from link:/assets/tools/rad-image[this link], or download it in your shell using curl:
+
+    curl -o rad-image https://radanalytics.io/assets/tools/rad-image
+    chmod +x rad-image
+    ./rad-image -h
+
+. Build images using the Spark distribution you've selected. This will create new `complete` tags on the radanaltyics.io image streams (it may take a while since it will build multiple images). Run `rad-image build -h` for additional information on the build command.
+
+    ./rad-image build https://archive.apache.org/dist/spark/spark-2.3.0/spark-2.3.0-bin-hadoop2.7.tgz
+
+. Make the radanalytics.io templates and configmaps use the `complete` tag
+
+    ./rad-image use complete
+
+. To use the default tags again, do this
+
+    ./rad-image use -d
+
+. To build images with a tag other than `complete`, specify the `-t TAG` option on the build command
```

Comment:
likewise, i think this step should be moved under the item for the initial build step. just to make it clear these are options.